### PR TITLE
Added RGBW (SK6812) support

### PR DESCRIPTION
**Pull request by Mysli0210**
I added rgbw support with white led calibration.
Works as intended with SK6812, though it might support other rgbw strips aswell.
The code can use some cleanup, as this is my first time using visual studio... and sucking at coding in general.

### CODE REVIEW

**Comment by fabsenet** [comment]
Very interesting, this is the first pull request for me and I already learned, you can reply via email to issues but not to pull requests!

What I noticed already:
- Please change strip type to enum and provide meaningful values. Simple true or false is neither descriptive nor extendable.
- Also please align your line endings to the project or whatever it is which make github think you changed complete files. The diff is broken.

Amazing, adrilight becomes more open source, I like it!

**Comment by Mysli0210** [comment]
Hi Fabian,

About enum, as I mentioned I have had like zero experience with visual
studio and never used enum, nor know what it means 😅

Also the difference is apparently when visual studio autoindents which
makes github think the whole file is changed.

Other than that I plan to make a white back light mode for when you just
want plain old white light for work and such.

And also a whitepoint calibration mode, where one side shows white with
only the white leds and the other showing "white" with only the rgb so you
can calibrate as good as easily possible.
Basically make them emit the same apparent colour.

Best Regards
Mysli

**Comment by Mysli0210** [comment]
Btw i tried to make a scroll menu using enum, but couldn't wrap my head
around it. so probably won't be able to solve that myself.

Den man. 15. jun. 2020 kl. 14.01 skrev Kasper Tulstrup <
kaspertulstrup@gmail.com>:
